packages/terra-toggle/src/terra-dev-site/doc/example/AnimatedToggle.jsx

```diff
@@ -19,7 +19,10 @@ class ToggleDefault extends React.Component {
     const { isOpen } = this.state;
     return (
       <div>
-        <IconInformation onClick={this.handleOnClick} />
+        <IconInformation onClick={this.handleOnClick} aria-expanded={isOpen} />
```

Comment:
Let's add the aria-controls attribute as well
```suggestion
        <IconInformation onClick={this.handleOnClick} aria-expanded={isOpen} aria-controls="toggleContainer" />
```

Reply:
Resolved in 3478f79